tests/utils/test_events.py

```diff
@@ -232,7 +232,9 @@ def test_completeness(self):
             k for k, v in iteritems(vars(zipline.utils.events))
             if isinstance(v, type) and
             issubclass(v, self.class_) and
-            v is not self.class_
+            v is not self.class_ and
+            # abstract class that doesn't need its own tests
+            v is not zipline.utils.events.TradingDayOfWeekRule
```

Comment:
Should we check for `abc` abstractness, as opposed to calling out this particular class?